cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/MockCruiseControl.java

```diff
@@ -196,8 +196,7 @@ public void setupCCRebalanceNotEnoughDataError(CruiseControlEndpoints endpoint)
                                 .withMethod("POST")
                                 .withQueryStringParameter(Parameter.param(CruiseControlParameters.JSON.toString(), "true"))
                                 .withQueryStringParameter(Parameter.param(CruiseControlParameters.DRY_RUN.toString(), "true|false"))
-                                .withQueryStringParameter(Parameter.param(CruiseControlParameters.VERBOSE.toString(), "true|false"))
-                                .withQueryStringParameter(buildBrokerIdParameter(endpoint))
```

Comment:
as mentioned offline, let's try to make this `setupCCxxxxxx` methods parameterized with an additional `String verbose` so that you can build the HttpRequest for the mock based on its value and if needed even the HttpResponse (as it happens for the `setupCCRebalanceResponse` more specifically).

Reply:
i have made this change now as we discussed, also tested nicely if eveything works